docs/readmes/lte/dev_notes.md

```diff
@@ -48,6 +48,72 @@ file is generated by mapping the configuration values from `gateway.mconfig` and
 `/etc/systemd/system` directory of the VM at the time of provisioning. You need
 to manually sync changes to these files between guest and host OS.
 
+- Systemd configuration files contain interdependencies of Magma services regarding the start, stop, and restart of services. Therefore, a systemd configuration contains several options to specify these dependencies.
+- The options `Before` and `After` define the order in which services are started and stopped. If a service gets started, the services listed in `Before` and `After` will also be started. If one of the other services gets stopped or its start fails, the service will be stopped. The following dependency graph depicts this order for Magma services.
```

Comment:
Can we link the official documentation here, i.e. https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Before= .